lib/ansible/modules/remote_management/manageiq/manageiq_provider.py

```diff
@@ -66,6 +66,12 @@
     default: null
     description: Tenant ID. defaults to None.
     version_added: "2.5"
+    aliases: [ keystone_v3_domain_id ]
```

Comment:
This might confusing to doc reader, documenting both with separate description may be better.  Not blocker, can fix when API is fixed.

Reply:
Ok, thanks again.